### ISSUE TITLE
[bib page] Unify dl formatting for fields with multiple definitions

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#183]
Bibliographic data is structured in a `<dl>` on the bib page. If there are multiple elements associated with a `<dt>`, these elements get handled differently. For some fields, the items are wrapped in a `<p>` and for some the items are in a `<ul>`. This should be normalized so that for all fields with multiple items (multiple subjects, multiple notes, etc), the items are structured in an `<ul>`. 

Excluding Availability, this is true for all fields, including Additional Authors (which currently concatenates authors with a comma.)

cc @EdwinGuzman

**Comment by ricardoom** [comment]
seems like some formatting got messed up in the message above...
